src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  font-family: 'Roboto';
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+
+
+.container {
+  width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  background: #FFFFFF;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+}
+.header-logo {
+  margin-left: 50px;
+  padding: 4px 0 0 0;
+}
+.header-navigation {
```

Same for the rest of the file
```suggestion
}

.header-navigation {
```